### ISSUE TITLE
Removing Quotes Case Different Results for Some Macros

### ISSUE THREAD

**Issue by kookma** [flibbles/tw5-uglify#5]
I use uglify 1.5.1 for compressing my TW, recently I noticed I get different results in a code uses `now` macro. See

```
<$let note-time=<<now 0hh:0mm>>
note-time2=<<now "0hh:0mm">>
>

* <<note-time>>
* <<note-time2>>

</$let>
```

This results in:

- 08:29, 25th September 2022
- 08:29

The original code is `<<now "0hh:0mm">>` after uglifying it is `<<now 0hh:0mm>>` note to disappeared `"`

**Comment by kookma** [comment]
I think the issue caused by `:`. The attribute value here is `0hh:0mm`.
May be Uglify should take care of attribute value with colon like those have spaces.

**Comment by flibbles** [comment]
This bug is worth making a release for. I must have forgotten to include the colon as a character that warrants quotation.

I'll push this fix soon.

**Comment by flibbles** [comment]
This should be fixed now as of Version 1.5.4
